Compilation fixes for the upcoming gcc 4.4 release.
Index: lyx-1.6.0/src/client/client.cpp
===================================================================
--- lyx-1.6.0.orig/src/client/client.cpp	2008-09-27 01:58:40.000000000 +0200
+++ lyx-1.6.0/src/client/client.cpp	2008-12-06 22:07:42.000000000 +0100
@@ -48,6 +48,7 @@
 #include <fcntl.h>
 
 #include <cerrno>
+#include <cstdio>
 #include <cstdlib>
 #include <string>
 #include <vector>
Index: lyx-1.6.0/src/frontends/qt4/GuiSymbols.cpp
===================================================================
--- lyx-1.6.0.orig/src/frontends/qt4/GuiSymbols.cpp	2008-05-23 10:43:07.000000000 +0200
+++ lyx-1.6.0/src/frontends/qt4/GuiSymbols.cpp	2008-12-06 22:07:42.000000000 +0100
@@ -31,6 +31,8 @@
 #include <QListWidgetItem>
 #include <QString>
 
+#include <cstdio>
+
 using namespace std;
 
 namespace lyx {
Index: lyx-1.6.0/src/support/lstrings.cpp
===================================================================
--- lyx-1.6.0.orig/src/support/lstrings.cpp	2008-10-09 09:21:48.000000000 +0200
+++ lyx-1.6.0/src/support/lstrings.cpp	2008-12-06 22:07:42.000000000 +0100
@@ -25,6 +25,7 @@
 #include <QVector>
 
 #include <algorithm>
+#include <cstdio>
 
 using namespace std;
 
